internal/overlord/servstate/handlers.go

```diff
@@ -406,7 +406,7 @@ func (s *serviceData) exited(waitErr error) error {
 		action, onType := getAction(s.config, waitErr == nil)
 		switch action {
 		case plan.ActionIgnore:
-			logger.Debugf("Service %q %s action is %q, transitioning to stopped state", s.config.Name, onType, action)
+			logger.Noticef("Service %q %s action is %q, transitioning to stopped state", s.config.Name, onType, action)
```

Comment:
This is so all three actions (restart, halt, ignore) have a similar log message with these details. This shouldn't happen often (only on service exit) and seems important.